nevergrad/functions/images/core.py

```diff
@@ -32,36 +34,54 @@ def __init__(self, problem_name: str = "recovering", index: int = 0) -> None:
         # Storing data necessary for the problem at hand.
         assert problem_name == "recovering"  # For the moment we have only this one.
         assert index == 0  # For the moment only 1 target.
-        #path = os.path.dirname(__file__) + "/headrgb_olivier.png"
+        # path = os.path.dirname(__file__) + "/headrgb_olivier.png"
         path = Path(__file__).with_name("headrgb_olivier.png")
         image = PIL.Image.open(path).resize((self.domain_shape[0], self.domain_shape[1]), PIL.Image.ANTIALIAS)
-        self.data = np.asarray(image)[:,:,:3]  # 4th Channel is pointless here, only 255.
+        self.data = np.asarray(image)[:, :, :3]  # 4th Channel is pointless here, only 255.
 
-        # if problem_name == "adversarial": 
-        #     assert index <= 100  # If we have that many target images.
-        #     self.data = ..... (here we load the imagee correspnding to index and problem_name; this is
-        #         # the attacked image.)
-
-        array = ng.p.Array(init=128 * np.ones(self.domain_shape), mutable_sigma=True,)
+        array = ng.p.Array(init=128 * np.ones(self.domain_shape), mutable_sigma=True, )
         array.set_mutation(sigma=35)
         array.set_bounds(lower=0, upper=255.99, method="clipping", full_range_sampling=True)
         max_size = ng.p.Scalar(lower=1, upper=200).set_integer_casting()
         array.set_recombination(ng.p.mutation.Crossover(axis=(0, 1), max_size=max_size)).set_name("")  # type: ignore
+
+        if problem_name == "adversarial":
```

Comment:
what is shared between the previous Image class and the new case? It seems to me there's hardly anything similar, in that case a different class would be much better